src/main/java/com/stripe/model/RecipientCardCollection.java

```diff
@@ -12,82 +12,137 @@
 import java.util.Map;
 
 public class RecipientCardCollection extends StripeCollection<Card> {
-  public RecipientCardCollection list(Map<String, Object> params)
-      throws AuthenticationException, InvalidRequestException,
-      APIConnectionException, CardException, APIException {
-    return list(params, null);
-  }
-
-  public RecipientCardCollection list(Map<String, Object> params,
-                    RequestOptions options) throws AuthenticationException,
-      InvalidRequestException, APIConnectionException, CardException,
-      APIException {
-    String url = String.format("%s%s", Stripe.getApiBase(), this.getURL());
-    return APIResource.requestCollection(url, params, RecipientCardCollection.class, options);
-  }
-
+  // <editor-fold desc="all">
+  /**
+   * List all recipient cards.
+   *
+   * @deprecated Use the {@link #list(Map)} method instead.
+   *     This method will be removed in the next major version.
+   */
   @Deprecated
   public RecipientCardCollection all(Map<String, Object> params)
       throws AuthenticationException, InvalidRequestException,
       APIConnectionException, CardException, APIException {
     return list(params, null);
   }
 
+  /**
+   * List all recipient cards.
+   *
+   * @deprecated Use the {@link #list(Map, RequestOptions)} method instead.
+   *     This method will be removed in the next major version.
+   */
   @Deprecated
   public RecipientCardCollection all(Map<String, Object> params,
-                     String apiKey) throws AuthenticationException,
+                     RequestOptions options) throws AuthenticationException,
       InvalidRequestException, APIConnectionException, CardException,
       APIException {
-    return list(params, RequestOptions.builder().setApiKey(apiKey).build());
+    return list(params, options);
   }
 
+  /**
+   * List all recipient cards.
+   *
+   * @deprecated Use the {@link #list(Map, RequestOptions)} method instead.
+   *     This method will be removed in the next major version.
+   */
   @Deprecated
   public RecipientCardCollection all(Map<String, Object> params,
-                     RequestOptions options) throws AuthenticationException,
+                     String apiKey) throws AuthenticationException,
       InvalidRequestException, APIConnectionException, CardException,
       APIException {
-    return list(params, options);
+    return list(params, RequestOptions.builder().setApiKey(apiKey).build());
   }
+  // </editor-fold>
 
-  public Card retrieve(String id)
+  // <editor-fold desc="create">
+  /**
+   * Create a recipient card.
+   */
+  public RecipientCardCollection create(Map<String, Object> params)
```

Comment:
same as above, we likely want to remove all of this in the next major version

Reply:
👍